pkg/kubelet/dockertools/manager.go

```diff
@@ -1804,7 +1804,7 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, runningPod kubecontainer.Pod, pod
 
 		// TODO(dawnchen): Check RestartPolicy.DelaySeconds before restart a container
 		namespaceMode := fmt.Sprintf("container:%v", podInfraContainerID)
-		_, err = dm.runContainerInPod(pod, container, namespaceMode, namespaceMode, getPidMode(pod))
+		_, err = dm.runContainerInPod(pod, container, namespaceMode, getIPCMode(pod, namespaceMode), getPidMode(pod))
```

Comment:
Nit.  I think while this technically works it doesn't follow the pattern that we take for the `netMode`.    For things that can point to another container (netMode & ipcMode - pidMode cannot) so far what we've done is configure the infra container and point the pod's containers to the infra container.

In other words - you can always use `namespaceMode` on this line which points to the infra container using "host".

@vishh - thoughts?  @simon3z is going on PTO, if this is a must fix for this to merge I can do it or I can do it as a quick follow up post-merge.